pcsx2/USB/usb-pad/dx/usb-pad-dx.cpp

```diff
@@ -57,8 +58,38 @@ namespace usb_pad
 				pad_copy_data(mType, buf, mWheelData);
 				return 5;
 			}
-
-			if (mType == WT_KEYBOARDMANIA_CONTROLLER)
```

Comment:
Was this a potential addition that was never completed?

Reply:
looks like a placeholder for the keyboardmania piano except there is already a place for keyboardmania below. i wonder if this would be more clear with a switch/case statement

Reply:
Still present at line 92